
`Error` 的子类，表明断言的失败。 
`assert` 模块抛出的所有错误都是 `AssertionError` 类的实例。

